app/src/main/res/values/strings.xml

```diff
@@ -393,6 +393,7 @@
   <string name="newmedicalhistory_no">No</string>
   <string name="newmedicalhistory_history">History</string>
   <string name="newmedicalhistory_diagnosis">Diagnosis (Required)</string>
+  <string name="medicalhistory_question_is_on_htn_treatment">BP medicines taken any time in the last month?</string>
```

Comment:
Can we avoid acronyms like "htn"? Let's use the full word, please?

Reply:
Approving this for now since it's urgent. We can rename this in another PR.

Reply:
Yep 👍🏾